Bottom: 29300d1844347314806a484be6c071c85ba34bfb
Top:    fda3912674c43eb3cfb9ce73f0a0a3895805a9d4
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-22 06:46:35 -0300

Refresh of wip-manual-document-mt-safety-string.patch

---

diff --git a/manual/string.texi b/manual/string.texi
index a75d298..6673a61 100644
--- a/manual/string.texi
+++ b/manual/string.texi
@@ -1159,6 +1159,11 @@ strings are written in into account.  To get that one has to use
 @comment string.h
 @comment BSD
 @deftypefun int strcasecmp (const char *@var{s1}, const char *@var{s2})
+@safety{@mtsafe{}@assafe{}@acsafe{}}
+@c Although this calls tolower multiple times, it's a macro, and
+@c strcasecmp is optimized so that the locale pointer is read only once.
+@c There are some asm implementations too, for which the single-read
+@c from locale TLS pointers also applies.
 This function is like @code{strcmp}, except that differences in case are
 ignored.  How uppercase and lowercase characters are related is
 determined by the currently selected locale.  In the standard @code{"C"}
